### DISCUSSION TITLE
Format 4 .aa chapter breaks

### DISCUSSION THREAD

**Discussion by Sailfish** [audiamus/AaxAudioConverter#46] [General]
I understand that AAC has some limitations when transcoding .aa (Format 4) files, however, is there a way of preserving the chapter breaks.

**Comment by audiamus** [comment]
It depends on several factors. 

First of all, the source .aa file must contain chapter marks which FFmpeg must be able to read. If the log at level 3 shows a list of chapters then this condition will be fulfilled.

For the conversion it depends somewhat on mode and destination file format. With chapter mode and split chapter mode you should get files divided by chapter and more, for both MP3 and M4A/M4B. For M4A/M4B you should also get embedded chapter marks for the other two conversion modes. Theoretically, for those conversion modes the embedded chapter marks should also appear in MP3 files but that doesn't always work and few players can read those, even if present.

However, positioning of the chapter marks will often not be as precise as in the original or as with AAX sources. I have not found the reason for this, it may have to do with FFmpeg.     

_PS: Using AAC as an acronym for AAX Audio Converter is confusing. AAC is commonly known to stand for the specific audio encoding (https://en.wikipedia.org/wiki/Advanced_Audio_Coding) which makes up the the audio stream in AAX and M4A/M4B._

**Reply by Sailfish** [reply]
As always, thanks for your quick and responsive reply. Since I am planning to batch convert over 1600 titles, I will experiment to see if I can find a sweet spot that provides for the most. Then, for those that don't work, I can always try an .aax version.

As an aside, my first go at it made the conversion of all of them in a bit over 4 hours, something easily scheduled overnight.

Note: I was aware of the other uses of the ambiguous ACC acronym but decided it was preferred over mangling "AAX Audio Converter" when typing; especially when needing to reference it multiple times. Would AAXAC be an acceptable abbreviation?

**Reply by audiamus** [reply]
> Would AAXAC be an acceptable abbreviation?

I'd think so, yes.
